doc/seg-schema.json

```diff
@@ -1,198 +1,201 @@
 {
-  "@context": "http://qiicr.org/dcmqi/contexts/dcmqi.jsonld",
-  "id": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/seg-schema.json#",
-  "$schema": "http://json-schema.org/draft-04/schema#",
-  "properties": {
-    "seriesAttributes": {
-      "type": "object",
-      "properties": {
-        "ContentCreatorName": {
-          "allOf": [
+  "@context":"http://qiicr.org/dcmqi/contexts/dcmqi.jsonld",
+  "id":"https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/seg-schema.json#",
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "properties":{
+    "seriesAttributes":{
+      "type":"object",
+      "properties":{
+        "ContentCreatorName":{
+          "allOf":[
             {
-              "$ref": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/PN"
+              "$ref":"https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/PN"
```

Comment:
It's just what the formatter I used generated: https://jsonformatter.curiousconcept.com/

I will run in http://www.jsonschemavalidator.net/ and update